rest/data/patch.go

```diff
@@ -213,17 +213,6 @@ func GetPatchRawPatches(patchID string) (map[string]string, error) {
 		}
 	}
 
-	if patchDoc.IsParent() {
```

Comment:
Oh yeah sorry -- I noticed in debugging something else that this doesn't matter, since raw doesn't return a patch doc. Meant to include in that ticket but I guess I got them mixed up.